sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionPumpManager.java

```diff
@@ -303,11 +316,24 @@ private void processEvents(PartitionContext partitionContext, PartitionProcessor
         }
     }
 
-    Map<String, EventHubConsumerAsyncClient> getPartitionPumps() {
+    Map<String, PartitionPump> getPartitionPumps() {
         return this.partitionPumps;
     }
 
-    private void handleError(PartitionOwnership claimedOwnership, EventHubConsumerAsyncClient eventHubConsumer,
+    class OnDropped implements Consumer<Object> {
+
+        private final String partitionId;
+        OnDropped(String partitionId) {
+            this.partitionId = partitionId;
+        }
+
+        @Override
+        public void accept(Object o) {
+            logger.warning("partitionId[{}] Dropped object: {}", partitionId, o);
```

Comment:
Why does this warning mean and what action can the user take? It might be better to just log as verbose as users may not be able to understand what dropped object means in their logs.

Reply:
Oh yes, I had that in for logging. Let me remove it.